NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# Jf create comment

#### Changes Made
1. Added Comment.cs, CommentRepository.cs, CommentController.cs, ICommentController.cs to server side
2. Added CommentForm.js and CommentManager.js to client side
3. modified PostDetails.js and ApplicationViews.js
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jf-commentCreate
    ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > When logged in as an authenticated user, navigate to the detail page for a post
    > There should be an 'Add Comment' button at the bottom of the page
    > Click on the button, and you should be redirected to a Comment Create form
    > The submit button should be disabled until both fields have text in them
    > After entering in text into the fields and clicking submit, the user should be taken back to the homepage
    > In order to test, you must check your actual database in VS to see if the new comment was created, rendering comments on the detail page will be my next issue

6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

closes #28
